libpal/wrmsr_x64.asm

```diff
@@ -0,0 +1,15 @@
+.code
+
+pal_execute_wrmsr proc
+    ; The address is already loaded in rcx because
+    ; it is the first integer argument. The value to
+    ; write is in rdx. We need that to be in edx:eax.
+    mov rax, rdx;
+    shr rdx, 032h;
```

Comment:
Remove the 'h' from 32h or change 32 to 20

Reply:
Personally I prefer to prefix hex values with '0x'

Reply:
> Personally I prefer to prefix hex values with '0x'

That's not supported by MASM AFAICT (hence the 'h' suffix). But in any case, you're right this is the wrong literal.